python/servo/testing_commands.py

```diff
@@ -57,6 +57,22 @@
 TEST_SUITES_BY_PREFIX = {path: k for k, v in TEST_SUITES.items() if "paths" in v for path in v["paths"]}
 
 
+def taplo(write: bool = False) -> int:
+    taplo = shutil.which("taplo")
+    if (taplo is not None):
```

Comment:
I would use an early return here.

```python
if not shutil.which("taplo"):
    print("Taplo is not installed.")
    print("It should be installed using `./mach bootstrap`, \
              but it can be installed manually using `cargo install taplo-cli --locked`")
    return 1
```